src/components/header/NavigationItems.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from 'react';
+
+import { useI18n } from '@smg-automotive/i18n-pkg';
+import { Box, Image, useMultiStyleConfig } from '@chakra-ui/react';
```

Comment:
Can you use our component `Box` instead importing it through '@chakra-ui/react'? Thanks